css/main.css

```diff
@@ -0,0 +1,142 @@
+* {
+	box-sizing: border-box;
+}
+
+
+
+.response {
+	/*margin: auto;
+	text-align: center;*/    
+    margin-left: 2em;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+	color: white;
```

Comment:
indentation should be consistent throughout. I'm only going to mention it here, but make sure you take a look at the entire file.